### DISCUSSION TITLE
posts/after-interviewing-software-engineers

### DISCUSSION THREAD

**Discussion by giscus** [teknologi-umum/blog#135] [General]
# posts/after-interviewing-software-engineers

The thoughts behind someone who might be interviewing you

https://teknologiumum.com/posts/after-interviewing-software-engineers

**Comment by mdestafadilah** [comment]
hallo sir, what's your programming langguage during the test live code? i have 2 times interviews with english language, not at all to live code :( just talking with HR person ... but the last  interviews, they need immediatly but i've not take it. i need notice 1 month in currently job.
